runtime/src/static_ids.rs

```diff
@@ -0,0 +1,14 @@
+use {
+    crate::{inline_spl_associated_token_account, inline_spl_token, inline_spl_token_2022},
+    solana_sdk::pubkey::Pubkey,
+};
+
+pub fn get_static_ids() -> Vec<Pubkey> {
```

Comment:
We might be able to make this a `lazy_static` so that it is constant, similar to [`FEATURE_NAMES` is here](https://github.com/solana-labs/solana/blob/04969a2282f639bd00b22143d3d1df59314bb5bc/sdk/src/feature_set.rs#L443)

Reply:
Good point. Made it use lazy_static, and we can just access the reference of the vector instead of having a function